cg/meta/workflow/fastq.py

```diff
@@ -299,28 +298,8 @@ def create_fastq_name(
         undetermined: Optional[str] = None,
         meta: Optional[str] = None,
     ) -> str:
-        """Name a FASTQ file following MIP conventions,
+        """Name a FASTQ file with standard conventions and
```

Comment:
Would it make sense to move this method into the base class `FastqHandler`? Might be out of scope for this PR, but it seems like this method might belong in the base class since it is very generic and could replace the existing implementation in the `FastqHandler` class

```python
    @staticmethod
    def create_fastq_name(
        lane: str,
        flowcell: str,
        sample: str,
        read: str,
        date: dt.datetime = DEFAULT_DATE_STR,
        index: str = DEFAULT_INDEX,
        undetermined: Optional[str] = None,
        meta: Optional[str] = None,
    ):
        raise NotImplementedError
```

Reply:
That would mean the `StandardFastqHandler` could be removed since it does not contain any additional methods or data and any references to it replaced with `FastqHandler`

Reply:
I like that. I thought about it but was unsure when I added it. But if there is no issues with that, I'll just replace it

Reply:
It seems more intuitive as well, I would expect any standard methods to be in the base class anyway and not in a separate class `StandardFastqHandler`. But there might be some good reason to structure it like this that I've missed 😄

Reply:
that's my fear ALL THE TIME 😬

Reply:
If it seems out of scope for this PR, then do it in a separate one or leave it 👍

Reply:
I've removed StandardFastqHandler and moved the method within the parent class